test/jdk/java/lang/Thread/virtual/ThreadPollOnYield.java

```diff
@@ -59,7 +62,7 @@ void testThreadYieldPolls() throws Exception {
         var vthread = Thread.ofVirtual().start(() -> {
             VThreadPinner.runPinned(() -> foo(done));
         });
-        Thread.sleep(5000);
+        started.await();
```

Comment:
I think this is okay. If the test is run directly with jtreg without libVThreadPinner.so then the virtual thread will throw and the main thread will wait indefinitely here, and the test will timeout.